robocorp-code/vscode-client/src/rcc.ts

```diff
@@ -215,6 +215,7 @@ export async function download(
             let response: XHRResponse = await xhr({
                 "url": url,
                 "onProgress": onProgress,
+                timeout: 5000 /* 5 sec */,
```

Comment:
I think that this timeout isn't ideal... the same function is used to download RCC (9MB), so, this could timeout too easily in valid cases depending on the speed connection.

So, if we need a limit it should be much bigger by default (or we should provide this as an argument and customize it just for this use case).